src/scantailor/filters/deskew/params.py

```
@@ -0,0 +1,81 @@
+This module defines the parameter structures for storing deskew settings
+on a per-page basis.
+"""
+
+from __future__ import annotations
+
+from enum import Enum
+
+from pydantic import BaseModel, Field
+
+
+class AutoManualMode(str, Enum):
```

Ruff (0.15.0)

[warning] Class AutoManualMode inherits from both `str` and `enum.Enum`

Inherit from `enum.StrEnum`

(UP042)